src/system/SearchModule/Block/SearchBlock.php

```diff
@@ -11,7 +11,6 @@
 
 namespace Zikula\SearchModule\Block;
 
-use ModUtil;
```

Comment:
no behavior diff AFAIK but easier to delete on merge to master without forgetting the import